From edbfafbd89409bfb1e4a4e3a9339c0b2de7435d6 Mon Sep 17 00:00:00 2001
From: Joanmarie Diggs <jdiggs@igalia.com>
Date: Thu, 27 Apr 2017 06:35:11 -0400
Subject: [PATCH] Return immediately in isLayoutOnly() if obj is dead

---
 src/orca/script_utilities.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: gnome-orca/src/orca/script_utilities.py
===================================================================
--- gnome-orca.orig/src/orca/script_utilities.py
+++ gnome-orca/src/orca/script_utilities.py
@@ -1159,7 +1159,7 @@ class Utilities:
 
         layoutOnly = False
 
-        if obj and self.isZombie(obj):
+        if self.isDead(obj) or self.isZombie(obj):
             return True
 
         try:
