From 382c5408afc7dd25f9b477a5e30c50ba917155c0 Mon Sep 17 00:00:00 2001
From: Joanmarie Diggs <jdiggs@igalia.com>
Date: Sat, 29 Apr 2017 11:28:17 -0400
Subject: [PATCH] Add check for dead accessibles before attempting to generate
 presentation

---
 src/orca/generator.py | 6 ++++++
 1 file changed, 6 insertions(+)

Index: gnome-orca/src/orca/generator.py
===================================================================
--- gnome-orca.orig/src/orca/generator.py
+++ gnome-orca/src/orca/generator.py
@@ -178,6 +178,12 @@ class Generator:
         - forceTutorial: boolean (default=False) which says if we
           should force a tutorial to be spoken or not
         """
+
+        if self._script.utilities.isDead(obj):
+            msg = 'ERROR: Cannot generate presentation dead obj'
+            debug.println(debug.LEVEL_INFO, msg, True)
+            return []
+
         startTime = time.time()
         result = []
         globalsDict = {}
