commit afe1d21742098871237c1e6675082ed42f099739
Author: Willie Walker <william.walker@sun.com>
Date:   Tue Jul 14 17:37:33 2009 -0400

    Remove errant comma at end of line

diff --git a/src/orca/formatting.py b/src/orca/formatting.py
index 40f633e..4fa0b45 100644
--- a/src/orca/formatting.py
+++ b/src/orca/formatting.py
@@ -565,7 +565,7 @@ if settings.useExperimentalSpeechProsody:
     formatting['speech'][pyatspi.ROLE_TERMINAL]['detailedWhereAmI'] = \
         'label + readOnly + pause + textRole + pause + textContentWithAttributes + anyTextSelection + ' + MNEMONIC + ' + ' + TUTORIAL
     formatting['speech'][pyatspi.ROLE_TEXT]['focused'] = \
-        'labelOrName + readOnly + textRole + pause + currentLineText + allTextSelection',
+        'labelOrName + readOnly + textRole + pause + currentLineText + allTextSelection'
     formatting['speech'][pyatspi.ROLE_TEXT]['unfocused'] = \
         'labelOrName + readOnly + textRole + pause + currentLineText + allTextSelection + ' + MNEMONIC
     formatting['speech'][pyatspi.ROLE_TEXT]['basicWhereAmI'] = \
