--- orca-3.0.0/src/orca/orca.in.orig	2011-04-04 22:32:01.000000000 +0200
+++ orca-3.0.0/src/orca/orca.in	2011-04-10 20:48:01.034458674 +0200
@@ -130,7 +130,7 @@
 # accessible login.  If we're running as gdm, we assume everything is
 # all set and we don't need to muck around.
 #
-if [ "x$LOGNAME" != "xgdm" ] ; then
+if [ "x$LOGNAME" != "xDebian-gdm" ] ; then
     ACCESSIBILITY_ENABLED=`gsettings \
         get org.gnome.desktop.interface toolkit-accessibility`
     if [ "x$ACCESSIBILITY_ENABLED" != "xtrue" ] ; then
