--- orca-3.0.2/src/orca/orca.in.orig	2011-05-24 01:15:30.000000000 +0200
+++ orca-3.0.2/src/orca/orca.in	2011-07-29 14:19:37.000000000 +0200
@@ -130,7 +130,7 @@
 # accessible login.  If we're running as gdm, we assume everything is
 # all set and we don't need to muck around.
 #
-if [ "x$LOGNAME" != "xgdm" ] ; then
+if [ "x$LOGNAME" != "xDebian-gdm" ] ; then
     ACCESSIBILITY_ENABLED=`gsettings \
         get org.gnome.desktop.interface toolkit-accessibility`
     if [[ "x$ACCESSIBILITY_ENABLED" != "xtrue" && \
