From: Joanmarie Diggs <jdiggs@igalia.com>
Date: Tue, 24 Apr 2018 11:41:30 -0400
Subject: Fix brlmon regression introduced by 71b63379f

---
 src/orca/braille.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/orca/braille.py b/src/orca/braille.py
index 2974108..716f825 100644
--- a/src/orca/braille.py
+++ b/src/orca/braille.py
@@ -1093,7 +1093,8 @@ def refresh(panToCursor=True,
     if stopFlash:
         killFlash(restoreSaved=False)
 
-    if not _settingsManager.getSetting('enableBraille'):
+    if not _settingsManager.getSetting('enableBraille') \
+       and not _settingsManager.getSetting('enableBrailleMonitor'):
         if _brlAPIRunning:
             try:
                 _brlAPI.writeText("", 0)
