diff --git a/src/orca/script_manager.py b/src/orca/script_manager.py
index 4078c27..5f0a9f8 100644
--- a/src/orca/script_manager.py
+++ b/src/orca/script_manager.py
@@ -60,6 +60,7 @@ class ScriptManager:
              'empathy-chat':     'empathy',
              'gnome-calculator': 'gcalctool',
              'marco':            'metacity',
+             'mate-notification-daemon': 'notification-daemon',
              'Nereid':           'Banshee',
              'vte':              'gnome-terminal',
              'gnome-terminal-server': 'gnome-terminal'}
